cupy/cuda/__init__.py

```diff
@@ -51,6 +51,12 @@
 except ImportError:
     nccl_enabled = False
 
+try:
+    from cupy_backends.cuda.libs import cudnn  # NOQA
```

Comment:
This is not needed as cudnn is not exposed here.
https://github.com/cupy/cupy/blob/v8.0.0b4/cupy/cuda/__init__.py

Reply:
Previous versions had `import cupy.cuda.cudnn as cudnn` as a valid path.
I think we should add a `cupy.cuda.cudnn` file that imports `cupy_backends.cuda.libs.cudnn` not to break compatibility?